rpy2_arrow/pyarrow_rarrow.py

```diff
@@ -191,28 +191,37 @@ def rarrow_to_py_field(
     return pyarrow.lib.Field._import_from_c(_pyarrow_ptr(schema_ptr))
```

Comment:
nit: Cannot leave comments on related but unchanged code, so adding the comment here:

I find `_pyarrow_ptr` to be a bit misleading as a name; perhaps `_c_ptr_to_int` or something similar?

Reply:
Fair request. Changed.